test/case/ietf_routing/ospf_basic/test.py

```diff
@@ -56,7 +56,22 @@ def config_target1(target, data, link):
     target.put_config_dict("ietf-routing", {
         "routing": {
             "control-plane-protocols": {
-                "control-plane-protocol": [{
+                "control-plane-protocol": [
```

Comment:
Not related to this line, but if possible, could you add some hints in the initial description on specific functionality tested? E.g., 
"and that they exchange routes" => "and that they exchange routes (regular and redistributed static/connected)"